CorpusCallosum/fastsurfer_cc.py

```diff
@@ -0,0 +1,856 @@
+
+logger = logging.get_logger(__name__)
+
+
+def options_parse() -> argparse.Namespace:
+    """Parse command line arguments for the pipeline."""
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--in_mri",
+        type=str,
+        required=False,
+        help="Input MRI file path. If not provided, defaults to subject_dir/mri/orig.mgz",
+    )
+    parser.add_argument(
+        "--cpu",
+        action="store_true",
+        help="Force CPU usage even when CUDA is available",
+    )
+    parser.add_argument(
+        "--aseg",
+        type=str,
+        required=False,
+        help="Input segmentation file path. If not provided, defaults to subject_dir/mri/aparc.DKTatlas+aseg.deep.mgz",
+    )
+    parser.add_argument(
+        "--subject_dir",
+        type=str,
+        required=False,
+        help="Subject directory containing standard FreeSurfer structure. "
+        "Required if --in_mri and --aseg are not both provided.",
+        default=None,
+    )
+    parser.add_argument("--debug_output_dir", type=str, required=False, default=None,
+                        help="Directory for debug output (default: subject_dir/qc_snapshots)")
+    parser.add_argument(
+        "--num_thickness_points", type=int, default=100, help="Number of points for thickness estimation."
+    )
+    parser.add_argument(
+        "--subdivisions",
+        type=float,
+        nargs="+",
+        default=[1/6, 1/2, 2/3, 3/4],
+        help="List of subdivision fractions for the corpus callosum subsegmentation.",
+    )
+    parser.add_argument(
+        "--subdivision_method",
+        type=str,
+        default="shape",
+        help="Method for contour subdivision. \
+                        Options: shape (Intercallosal subdivision perpendicular to intercallosal line), vertical \
+                        (orthogonal to the most anterior and posterior points in the AC/PC standardized CC contour), \
+                        angular (subdivision based on equally spaced angles, as proposed by Hampel and colleagues), \
+                        eigenvector (primary direction, same as FreeSurfers mri_cc)",
+        choices=["shape", "vertical", "angular", "eigenvector"],
+    )
+    parser.add_argument(
+        "--contour_smoothing",
+        type=float,
+        default=5,
+        help="Window size for smoothing during contour detection. Default is 5, higher values mean a smoother"
+        "outline, at the cost of precision.",
+    )
+    parser.add_argument(
+        "--slice_selection",
+        type=str,
+        default="all",
+        help="Which slices to process. Options: 'middle' (default), 'all', or a specific slice number.",
+    )
+    parser.add_argument(
+        "--upright_volume_path",
+        type=str,
+        help=f"Path for upright volume output (default: subject_dir/{STANDARD_OUTPUT_PATHS['upright_volume']})",
+        default=None,
+    )
+    parser.add_argument(
+        "--segmentation_path",
```

Comment:
```suggestion
        "--seg",
        dest="segmentation_path",
```
Other FastSurfer interfaces have also used `segfile` (e.g. `--aseg_segfile`) -- this is almost an exception to the no suffix rule.